src/utils.js

```diff
@@ -86,21 +112,24 @@ const list = async (db, encKey, prefix) => {
   if (!db) throw new MasqError(ERRORS.NO_DB)
   if (!encKey) throw new MasqError(ERRORS.NO_ENCRYPTION_KEY)
 
-  const list = await db.listAsync(prefix)
+  const _prefix = prefix ? hashKey(prefix) : '/'
+  const list = await db.listAsync(_prefix)
   if (list.length === 1 && list[0].key === '' && list[0].value === null) {
     return {}
   }
-
-  const decList = await Promise.all(list.map(async (elt) => ({
-    key: elt.key,
-    value: await decrypt(encKey, elt.value)
-  })))
+  const decList = await Promise.all(list.map(async (elt) => {
+    const dec = await decrypt(encKey, elt.value)
+    return {
+      key: dec.key,
+      value: dec.value
+    }
```

Comment:
Can't you return dec directly ?

Reply:
does dec contains more values than just `key` and `value` ?